app/src/main/res/xml/keyboard_qwerty_german.xml

```diff
@@ -0,0 +1,65 @@
+        <Key android:codes="114" android:keyLabel="r" android:popupCharacters="rŕɍřŗṙ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="116" android:keyLabel="t" android:popupCharacters="tťṫţṭțṱṯ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="122" android:keyLabel="z" android:popupCharacters="zźẑžżẓ" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="117" android:keyLabel="u" android:popupCharacters="uŭʉụüǜ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="105" android:keyLabel="i" android:popupCharacters="iíìĭîǐï" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="111" android:keyLabel="o" android:popupCharacters="oøǿöȫóò" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="112" android:keyLabel="p" android:popupCharacters="pṕṗᵽ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:keyOutputText="ü" android:keyLabel="ü" android:popupCharacters="uŭʉụüǜ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="95" android:keyLabel="_"/>
+        <Key android:codes="-5" android:keyIcon="@drawable/ic_icon_keyboard_backspace" android:isRepeatable="true" android:keyWidth="@dimen/keyboard_key_backspace_width" android:keyEdgeFlags="right" />
+    </Row>
+
+    <Row>
+        <Key android:codes="97" android:keyLabel="a" android:popupCharacters="aáăäâą" android:popupKeyboard="@xml/keyboard_popup" android:keyEdgeFlags="left" android:horizontalGap="@dimen/keyboard_left_margin" />
+        <Key android:codes="115" android:keyLabel="s" android:popupCharacters="sśṥŝšṧṡş" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="100" android:keyLabel="d" android:popupCharacters="dđɗḋḍḑḓ" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="102" android:keyLabel="f" android:popupCharacters="fƒḟ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="103" android:keyLabel="g" android:popupCharacters="gǵǥĝǧğģ" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="104" android:keyLabel="h" android:popupCharacters="hĥȟħḩⱨẖ" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="106" android:keyLabel="j" android:popupCharacters="jĵɈɉj̇̃" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="107" android:keyLabel="k" android:popupCharacters="kƙꝁḱǩḳķ" android:popupKeyboard="@xml/keyboard_popup"/>
+        <Key android:codes="108" android:keyLabel="l" android:popupCharacters="lĺłľḹl̃ļ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:keyOutputText="ö" android:keyLabel="ö" android:popupCharacters="oøǿöȫóò" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:keyOutputText="ä" android:keyLabel="ä" android:popupCharacters="aáăäâą" android:popupKeyboard="@xml/keyboard_popup" />
```

Comment:
after this key, it looks like there should be another key before Enter:

![image](https://user-images.githubusercontent.com/203725/57397664-0da33e00-7182-11e9-8c48-ae4ad5927873.png)

![image](https://user-images.githubusercontent.com/203725/57397699-198f0000-7182-11e9-9212-42dfd18ea5ff.png)